tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/group_445_event_445.pysnap

```diff
@@ -447,7 +447,7 @@ system:
               "crashpad_client_win.cc"
             function*
               "crashpad::`anonymous namespace'::HandleAbortSignal"
-        type (ignored because exception is synthetic)
+        type*
```

Comment:
I wonder if we should write a hint into the system's type that we *don't* ignore it because X.

Reply:
I'm having a hard time coming up with a good hint here -- it should be helpful to the user but the reason we don't ignore the component is an implementation detail.